cves/CVE-2015-3982.yml

```diff
@@ -194,8 +210,10 @@ ipc:
     software system reads is another form of IPC.
 
     Answer should be boolean.
-  answer:
-  note:
+  answer: true
+  note: |
+    Because this vulnerability gave user unauthorized access using the session key, you are able to access features that you
```

Comment:
I assume it would also be IPC because this method is supposed to update the database.